bindings_ffi/src/mls.rs

```diff
@@ -1208,21 +1192,38 @@ impl FfiConversations {
             _ => None,
         };
 
-        let convo = if account_identities.is_empty() {
-            let group = self
-                .inner_client
-                .create_group(group_permissions, metadata_options)?;
-            group.sync().await?;
-            group
-        } else {
-            self.inner_client
-                .create_group_with_members(&account_identities, group_permissions, metadata_options)
-                .await?
-        };
+        let convo = self
+            .inner_client
+            .create_group(group_permissions, metadata_options)?;
```

Comment:
Does adding metadata options not create a commit? cc @cameronvoell

Reply:
It's probably okay either way. Regardless of if a commit was queued or not, what we really want is for the method to return immediately so that integrators have a group ID. Given the lack of `await`, the network requests are probably only going out when `sync()` is being called.

We should give clear guidance to integrators on what to do though, and validate it matches their intent. When a group is created in the UI, this method can be called to return a group ID synchronously, and either `add_members()` or `sync()` can be called afterwards to add members or sync your own installations respectively, and the UI shouldn't block on the second method returning.